app.rb

```diff
@@ -0,0 +1,40 @@
+require 'sinatra/base'
+require_relative './lib/peeps'
+require './lib/users'
+require_relative './database_connection_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    @user = Users.find(session[:user_id])
+    erb :'index'
+  end
+
+  get '/peeps' do
+    @peeps = Peeps.all.sort_by { |a| -a.id.to_i }
+    erb :'peeps/peeps'
+  end
+
+  post '/peeps-add' do
+    Peeps.create(message: params[:peep])
+    redirect '/peeps'
+  end
+
+  get '/peeps/new' do
+    erb :'/peeps/new_peep'
+  end
+
+  get '/users/new' do
+    Users.create(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
I just wanted to check with you why you called Users.create here. My understanding is it can be called in post '/user-add' for example as once the user loads this page (get), they will enter username, email and password which during a post can be used to create a User account for them like you did below. However at get '/users/new' they are loading the page so haven't entered any params for username, email and password.